common/src/com/thoughtworks/go/config/materials/PackageMaterial.java

```diff
@@ -118,7 +118,7 @@ public String getFolder() {
     }
 
     @Override
-    public void updateTo(ProcessOutputStreamConsumer outputStreamConsumer, File baseDir, RevisionContext revisionContext, SubprocessExecutionContext execCtx) {
+    public void updateTo(ProcessOutputStreamConsumer outputStreamConsumer, File baseDir, boolean ignoreDestination, RevisionContext revisionContext, SubprocessExecutionContext execCtx) {
```

Comment:
With this change in https://github.com/gocd/gocd/commit/11e36ea7c1a9ee1cf5bcfcc4eedbdf6bf076c8af

You don't need the extra parameter the ```SubprocessExecutionContext``` has the ```isServer``` method that can be used to add this behaviour.